guides/common/modules/proc_installing-ostree-content-from-server.adoc

```diff
@@ -0,0 +1,23 @@
+[id="Installing_OSTree_Content_from_{project-context}_Server_{context}"]
+= Installing OSTree content from {ProjectServerTitle}
+
+OSTree content from {Project} on hosts is managed by Subscription manager and can be accessed using `rpm-ostree`.
+
+.Prerequisites
+* You have uploaded or synchronized a OSTree head to {Project}.
```

Comment:
How? What exactly does it mean? Can we refer to the import procedure or something?

Reply:
I took the Python docs as reference here, so this is how this ended up like here.
But yes sure, I will extend it 👍🏻